### ISSUE TITLE
### Day 6 - Accessibility

### ISSUE THREAD

**Issue by krdurrett** [krdurrett/Overlook#13]
Note!If you haven’t already, focus on accessibility at this point. Before moving to iteration 3, please create a branch and push it up to GH so instructors can run Lighthouse and check your dashboard for it’s accessibility audit.
Accessibility
- [x] Accessibility audits should be at 100% for the dashboard.
- [x] A user should be able to interact with all  functionality of your application by tabbing through it, no use of the trackpad
- [x] ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
